donkeycar/parts/gps.py

```diff
@@ -178,7 +181,12 @@ def getGpsPosition(line, debug=False):
     nmea_msg = line[1:-3]      # msg without $ and *## checksum
     nmea_parts = nmea_msg.split(",")
     message = nmea_parts[0]
-    if (message == "GPRMC") or (message == "GNRMC"):   
+
+    if (message == "GPRMC") or (message == "GNRMC"):
+        if True or debug:
```

Comment:
This is always True - why do we need the `if`. Also we are printing, not logging into the shell, is this intended. I would expect to just have:
```python
logger.debug(line)
```
so if we set global logging to Debug, it will log, otherwise is won't.

Reply:
I deleted it; it was left over from finding the problem.